app/src/main/java/com/orange/ods/app/ui/AppNavigation.kt

```diff
@@ -10,11 +10,51 @@
 
 package com.orange.ods.app.ui
 
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
 import androidx.lifecycle.Lifecycle
 import androidx.navigation.NavBackStackEntry
 import androidx.navigation.NavController
 import androidx.navigation.NavDestination
 import androidx.navigation.NavGraph
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.rememberNavController
+
+@Composable
+fun rememberAppNavigationState(navController: NavHostController = rememberNavController()) = remember(navController) { AppNavigationState(navController) }
+
+class AppNavigationState(val navController: NavHostController) {
+    val previousRoute: String?
+        @Composable get() = navController.previousBackStackEntry?.destination?.route
+
+    val currentScreenRoute: String?
```

Comment:
We now have `previousRoute` and `currentScreenRoute`. I think both names should contain (or not) `Screen`.